config/services.yml

```diff
@@ -66,6 +70,7 @@ services:
   Symfony\Component\Console\Output\ConsoleOutput: ~
   Symfony\Component\Console\Output\OutputInterface:
     alias: Symfony\Component\Console\Output\ConsoleOutput
+  Symfony\Component\EventDispatcher\EventDispatcher: ~
```

Comment:
Might have been part of it, but it wasn't the only problem, the kernel never would have included the dispatcher in the compiler pass since it was only looking for commands.